java/src/jmri/jmrit/audio/swing/AudioSourceFrame.java

```diff
@@ -417,7 +409,8 @@ private boolean applyPressed(ActionEvent e) {
     }
 
     private static int nextCounter() {
-        return counter++;
+        counter++;
+        return counter;
     }
```

Comment:
I'm not sure, but it seems to me that `counter` should be set to 0 instead of 1 on line 44 due to this change.